src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -63,7 +63,7 @@
     :status-tag
     (let [community-id (rf/sub [:community-id-by-chat-id chat-id])]
       (conj units
-            [rn/text
+            [quo/text
```

Comment:
im not sure rn/text can be just replaced by quo/text

Reply:
me neither, can be addressed later anyway